usage/camp/src/main/java/io/brooklyn/camp/brooklyn/YamlLauncherAbstract.java

```diff
@@ -79,8 +80,8 @@ public Application launchAppYaml(String url) {
 
     public Application launchAppYaml(String url, boolean waitForTasksToComplete) {
         try {
-            Reader input = Streams.reader(new ResourceUtils(this).getResourceFromUrl(url));
-            Application app = launchAppYaml(input, waitForTasksToComplete);
```

Comment:
This is the wrong method to update, replace the one that was called here. For a global check if the patch was successful check for references like `registerDeploymentPlan`, `instantiate`, they should we restricted to the `EntityManagementUtils` class only (outside of camp projects).